### PR TITLE
Version code

### PR DESCRIPTION
**Pull request by bmarty**
Fix downgrade issue due to version code recent change (in https://github.com/vector-im/element-x-android/pull/932).
Also:
- Firebase will get the app bundle instead of the APK.
- Maestro will get the x86_64 APK instread of the universal one.

### CODE REVIEW

**Review by csmith — APPROVED** [review]
LGTM, couple of suggestions but they can be done later if necessary :)

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -44,7 +44,9 @@ jobs:
           FIREBASE_TOKEN: ${{ secrets.ELEMENT_ANDROID_NIGHTLY_FIREBASE_TOKEN }}
       - name: Additionally upload Nightly APK to browserstack for testing
         continue-on-error: true # don't block anything by this upload failing (for now)
-        run: curl -u "$BROWSERSTACK_USERNAME:$BROWSERSTACK_PASSWORD" -X POST "https://api-cloud.browserstack.com/app-automate/upload" -F "file=@app/build/outputs/apk/nightly/app-universal-nightly.apk" -F "custom_id=element-x-android-nightly"
+        run: |
+          ./gradlew assembleNightly $CI_GRADLE_ARG_PROPERTIES
```
[the remark this answers is not in the document]
Comment (bmarty) [reply .github/workflows/nightly.yml:48]: Ah yes, good point, added.

File: plugins/src/main/kotlin/Versions.kt [hunk 2]
```diff
@@ -17,6 +17,33 @@
...
+ * Current version of Element Android (at some point EAx will replace this app) (v1.6.3)
+ * ----40_106_03a where a stands for the architecture: 1, 2, 3, 4 and 0 for the universal APK
+ * Current version of EAx distributed with Firebase app distribution:
+ * ----1_002_000
+ * Latest version of EAx distributed with Firebase app distribution (downgrading, so that's a problem)
+ * -------10_200
+ * Version when running the current debug build
+ * -------10_200
+ *
+ * So adding 4_000_000 to the current version Code computed here should be fine, we will have:
```
[the remark this answers is not in the document]
Comment (bmarty) [reply plugins/src/main/kotlin/Versions.kt:38]: Yes, you're right! Done.

**Comment by bmarty** [comment]
Oups, I have merged without pushing my latest commits, I will push them on develop.

**Comment by bmarty** [comment]
There is an issue with Firebase Distribution that I am investigating:

```
 > Task :app:appDistributionUploadNightly FAILED
Using AAB path specified by the artifactPath parameter in your app's build.gradle: /home/runner/work/element-x-android/element-x-android/app/build/outputs/bundle/nightly/app-nightly.aab.
Uploading AAB to Firebase App Distribution...
Using credentials token specified by environment variable FIREBASE_TOKEN

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:appDistributionUploadNightly'.
> App Distribution failed to process the AAB: This project is not linked to a Google Play account.
```

Good news is that Mastro is happy with the x86_64 APK.

**Comment by bmarty** [comment]
When doing it manually, I've got:

image

**Comment by bmarty** [comment]
App on Firebase is now linked to GooglePlay, to be able to release nightly build as AAB

image

**Comment by bmarty** [comment]
So reverting this change in #945 to unblock the nightly flow.

**Pull request merged**